blocks/section-background-image-gradient/section-background-image-gradient.css

```diff
@@ -0,0 +1,63 @@
+
+.section > .section-background-image-gradient-wrapper,
+.section > .section-background-image-gradient-wrapper > .section-background-image-gradient.block {
+    padding: 0;
+    margin: 0;
+    max-width: 100%;
+    width: 100%;
+    position: relative;
+}
+
+.section-background-image-gradient > div:first-of-type {
+    position: absolute;
+    height: 200px;
+    width: 100%;
+    z-index: 1;
+    background: linear-gradient(
+            180deg,
+            rgb(235 237 238 / 100%) 0%,
```

Comment:
@bstopp, let me know if you'd like me to match these colors with one of the gray color values we already have, and I can translate that and assign to a variable if needed.